src/lambda/utils/contextTable.ts

```diff
@@ -46,10 +46,12 @@ export const createContextTable = async (target: CrawlInputWithId, contextTableN
         KeyType: 'RANGE',
       },
     ],
+    BillingMode: 'PAY_PER_REQUEST'
+    /*,
```

Comment:
Again let's get rid of commented code :)

Reply:
Fixed